### PR TITLE
[res] More ONNX regression test of CONV+BN

### PR DESCRIPTION
**Pull request by seanshpark**
This will add more ONNX transformatio regression test models of CONV+BN.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
From draft #7394

File: res/TensorFlowLiteRecipes/REGRESS_ONNX_Conv_BN_Relu6_001/test.rule [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To check ONNX conversion is OK
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "CONV_EXIST"              $(op_count CONV_2D) '=' 1
+RULE    "NO_TRANSPOSE"            $(op_count TRANSPOSE) '=' 0
+RULE    "NO_ADD"                  $(op_count ADD) '=' 0
+RULE    "NO_MUL"                  $(op_count MUL) '=' 0
+RULE    "NO_RELU6"                $(op_count RELU6) '=' 0
```
Comment (jinevening) [inline res/TensorFlowLiteRecipes/REGRESS_ONNX_Conv_BN_Relu6_001/test.rule:9]: Relu6 may exist ? Min-Max will be converted to Relu6.

Comment (seanshpark) [reply res/TensorFlowLiteRecipes/REGRESS_ONNX_Conv_BN_Relu6_001/test.rule:9]: It should be fused... `fuse_activation_function` exist in dredd test.

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**